<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Google Data Qt Client - Blogger example</string>
  </property>
  <property name="locale">
   <locale language="Spanish" country="Spain"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <widget class="QFrame" name="header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/res/header.png);
border: 0px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>754</x>
       <y>10</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background:  #003366;
border: 0px;
color: #6699ff;</string>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="body">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>79</y>
      <width>800</width>
      <height>520</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/res/back.png);
border: 0px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
background-image: url(:/images/res/mainback.png);
}
QTabWidget::pane { /* The tab widget frame */
border-top: 2px solid #baa68e;
}
QTabWidget::tab-bar {
left: 15px; /* move to the right by 5px */
}
/* Style the tab using the tab sub-control. Note that it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
background: #e1d4c0;
border: 1.5px solid #baa68e;
border-bottom-color: #baa68e; /* same as the pane color */
min-width: 18ex;
padding: 5px;
}
QTabBar::tab:hover {
color: #ff6600; background: #f5ede3; 
}
QTabBar::tab:selected {
font:bold;
background: #f5ede3; 
border-color: #9B9B9B;
border-bottom-color:#baa68e;/* same as pane color */
}
QTabBar::tab:!selected {
margin-top: -1px; /* make non-selected tabs look smaller */
}
QTabBar::tab:selected {
/* expand/overlap to the left and right by 4px */
margin-left: -4px;
margin-right: -4px;
}
QTabBar::tab:first:selected {
margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
}
QTabBar::tab:last:selected {
margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
}
QTabBar::tab:only-one {
margin: 0; /* if there is only one tab, we don't want overlapping margins */
}


</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="entradasTab">
      <attribute name="title">
       <string>Entradas</string>
      </attribute>
      <widget class="QFrame" name="entradasFrame">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>10</y>
         <width>770</width>
         <height>400</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #f5ede3;
border: 4px solid #e1d4c0;
border-top-left-radius: 18px;
border-top-right-radius: 18px;
border-bottom-right-radius: 18px;
border-bottom-left-radius: 18px;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>25</y>
          <width>760</width>
          <height>350</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #fff;
border: 0px;
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;
</string>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::CustomDashLine</enum>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column/>
       </widget>
      </widget>
      <widget class="QPushButton" name="newButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>430</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #6699ff;
color: white; 
font:bold;
border-top-left-radius: 8px;
border-top-right-radius: 8px;
border-bottom-right-radius: 8px;
border-bottom-left-radius: 8px;
border: 1px solid #003399;
</string>
       </property>
       <property name="text">
        <string>NUEVA ENTRADA</string>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteSelectionButton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>430</y>
         <width>171</width>
         <height>27</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #ff6600;
color: white; 
font:bold;
border-top-left-radius: 8px;
border-top-right-radius: 8px;
border-bottom-right-radius: 8px;
border-bottom-left-radius: 8px;
border: 1px solid #993300;
</string>
       </property>
       <property name="text">
        <string>SUPRIMIR SELECCIÃ“N</string>
       </property>
      </widget>
      <widget class="QFrame" name="editPostFrame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>490</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="publishButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>430</y>
          <width>161</width>
          <height>27</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #6699ff;
color: white; 
font:bold;
border-top-left-radius: 8px;
border-top-right-radius: 8px;
border-bottom-right-radius: 8px;
border-bottom-left-radius: 8px;
border: 1px solid #003399;
</string>
        </property>
        <property name="text">
         <string>PUBLICAR ENTRADA</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cancelButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>430</y>
          <width>171</width>
          <height>27</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #ff6600;
color: white; 
font:bold;
border-top-left-radius: 8px;
border-top-right-radius: 8px;
border-bottom-right-radius: 8px;
border-bottom-left-radius: 8px;
border: 1px solid #993300;
</string>
        </property>
        <property name="text">
         <string>CANCELAR</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="titleTextEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>751</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #baa68e;
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-right-radius: 4px;
border-bottom-left-radius: 4px;
</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="contentcTextEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>751</width>
          <height>321</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #baa68e;
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-right-radius: 4px;
border-bottom-left-radius: 4px;
</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="comentariosTab">
      <attribute name="title">
       <string>Comentarios</string>
      </attribute>
      <widget class="QFrame" name="commentariosFrame">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>10</y>
         <width>770</width>
         <height>400</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #f5ede3;
border: 4px solid #e1d4c0;
border-top-left-radius: 18px;
border-top-right-radius: 18px;
border-bottom-right-radius: 18px;
border-bottom-left-radius: 18px;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QTableWidget" name="commentsTableWidget">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>25</y>
          <width>760</width>
          <height>350</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #fff;
border: 0px;
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;
</string>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::CustomDashLine</enum>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column/>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
